tests/test_storage/test_storage.py

```diff
@@ -106,6 +106,7 @@ def test__df_to_blob_call_args_2(
             url=blob_url,
             headers={"x-ms-blob-type": "BlockBlob"},
             data=ANY,
+            timeout=(10, None),
```

Comment:
Is this a total time to last byte? If so 10 second can be short for downloading larger blobs? I want to read in details on this tomorrow.

Reply:
But we need to have a timeout on the reply too? The example I shown where we connect, but never reply, should timeout too? Perhaps better to use this?
![image](https://github.com/4Subsea/drio-python/assets/88324093/b502dbfc-794a-430e-9e08-87d5914281b0)

Reply:
What woud that read timeout be in case of a `PUT` call towards Azure Blob with files up-to 4GB in size?

Reply:
20 mins? 1 hour?

Reply:
I am not sure, this is a tricky area. I need to read up on how they define timeouts in these libraries. Total time might not be easy to set for large files, but there could be some timing on the responses, as this is uploaded in chunks to the server.

Reply:
For the blobs, it might be beneficial to use the blob storage client from Azure anyway, because they might have solved some of this, or at least look into the source/docs on how they do this https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob#other-client--per-operation-configuration

Reply:
I think this is what I am looking for, the large upload is a lot of smaller packets
![image](https://github.com/4Subsea/drio-python/assets/88324093/54602d5d-fe1f-4c8e-8744-ed1f133d7505)

Reply:
Surely the large files must be uploaded in smaller increments, and not one massive POST or PUT operation of several GBs?